client/components/domains/domain-search-results/style.scss

```diff
@@ -61,7 +61,7 @@ body.is-section-signup {
 }
 
 .domain-search-results__domain-available-notice {
-	display: flex;
+	display: flex !important;
```

Comment:
I'm always a little wary when I see `! important` in CSS 😬 but it should be ok for now